triaging.rst

```diff
@@ -375,6 +375,17 @@ Various informational flags about the issue. Multiple values are possible.
 | easy          | Fixing the issue should not take longer than a day for     |
 |               | someone new to contributing to Python to solve.            |
 +---------------+------------------------------------------------------------+
+| easy (C)      | Fixing the issue should not take longer than a day for     |
+|               | someone new contributing to Python, focused on C.          |
++---------------+------------------------------------------------------------+
+| security_issue| The issue would fit as, or is related as a security issue. |
+|               | The main difference from "security" is that this is a      |
```

Comment:
"security" is out of context here, so maybe write something "The main difference from the "secutiry" issue type is..."

Reply:
Added both changes, I think also since there is no security keyword in the bug tracker, the main difference is that is a definite problem right? While the security type above it is more for reporting a possible issue.